views/facetsWithArticles/clusteredImages/four.hbs

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+	{{#> head title='Clustered Images - v4' subtitle='' }}
+	<style type="text/css">
+		main {
+		    width: 100%;
+		}
+		h2 {
+			color: #fff;
+		    width: 1000%;
+		    font-size: 8vw;
+		    text-shadow:
+			   -1px -1px 0 #000,  
+			    1px -1px 0 #000,
+			    -1px 1px 0 #000,
+			     1px 1px 0 #000;
+		}
+		button {
+			position: relative;
+			display: block;
+			margin-top: 10px;
+		}
+		.cluster {
+		    display: inline-block;
+		    width: 200px;
+		    height: 200px;
+		    position: relative;
+		}
+		.cluster--images {
+			position: absolute;
+			width: 100%;
+			height: 100%;
+		}
+		.cluster--images img {
+			position: absolute;
+			width: auto;
+			height: auto;
+			max-width: 100%;
+			left: 0px;
+    		top: 45px;
+		}
+		.circlemask {
+			clip-path: circle(72%);
+		}
+		.centerPos {
+			position: absolute;
+			margin: 0;
+		    top: 50%;
+		    left: 50%;
+		    transform: translate(-50%, -50%);
+		    text-align: center;
+		}
+		.controls {
+			position: absolute;
+			display: block;
+			padding: 10px;
+			top: 5px;
+			right: 5px;
+			width: auto;
+			height: auto;
+			font-size: 12px;	
+		}
+		.guideline {
+			border: 1px dashed black;
+		}
+	</style>
+	{{/head}}
+	<body>
+		<main class="centerPos">
+			{{#each data}}
+			<div class="cluster guideline">
+				<div class="cluster--images">
+				{{#each articles}}
+				{{#if images.0.url}}
+					<img class="" src="{{imgPath images.0.url}}" />
+				{{/if}}
+				{{/each}}
+				</div>
+				<h2 class="centerPos">{{facetName}}</h2>
+			</div>
+			{{/each}}
+		</main>
+		<fieldset class="controls">
+			<legend>Controls:</legend>
+			<label><input type="checkbox" name="mask" id="clipMask">Mask</label><br />
+			<label><input type="checkbox" name="guide" id="guideLine" checked>Guide</label>
+			<button>Refresh positions</button>
+		</fieldset>
+
+		<script type="text/javascript">
```

Comment:
Would be worth having a separate JS here. On a side note: not all browsers will support ES6 notation. Fine for Chrome demos, but something to bear in mind.

Reply:
Got it. Do you normally write client side JS without ES6 or use something like Babel?